manual/Website2/package.json

```diff
@@ -6,10 +6,11 @@
     "url": "https://github.com/faceyspacey/remixx.git"
   },
   "author": "Zack Jackson <zack@ScriptedAlchemy.com> (https://github.com/ScriptedAlchemy)",
+  "license": "GPL-3.0-only",
   "scripts": {
-    "manual:prod": "cd ../../ && yarn link  && cd manual/Website2 && yarn link  webpack-external-import && cross-env NODE_ENV=production node  node_modules/webpack-dev-server/bin/webpack-dev-server.js",
-    "manual:dev": "cd ../../ && yarn link  && cd manual/Website2 && yarn link  webpack-external-import && cross-env NODE_ENV=development node  node_modules/webpack-dev-server/bin/webpack-dev-server.js",
-    "manual:debug": "cd ../../ && yarn link  && cd manual/Website2 && yarn link  webpack-external-import && cross-env NODE_ENV=development node --inspect node_modules/webpack-dev-server/bin/webpack-dev-server.js"
```

Comment:
because of yarn workspaces, we can't guarantee the location - may be hoisted or not